ftag/flavours.yaml

```diff
@@ -94,6 +94,31 @@
   cuts: ["R10TruthLabel_R22v1 == 10"]
   colour: "#38761D"
   category: xbb
+- name: qcdbb
+  label: QCD->bb
+  cuts: ["R10TruthLabel_R22v1 == 10", "GhostBHadronsFinalCount == 2"]
+  colour: "red"
+  category: xbb
+- name: qcdnonbb
+  label: QCD
+  cuts: ["R10TruthLabel_R22v1 == 10", "GhostBHadronsFinalCount != 2"]
+  colour: "silver"
+  category: xbb
+- name: qcdbl
+  label: QCD
+  cuts: ["R10TruthLabel_R22v1 == 10", "GhostBHadronsFinalCount == 1"]
```

Comment:
This might give bc no?

Reply:
yes, so the idea was actually to have qcdbx (same below qcdxx) so x would be either light or c
hence the definitions
I think we only care that they remain mutually exclusive which still stands with these definitions 

If you find the names confusing, I can just rename them to qcdbx instead of qcdbl (would be my preference)
If instead you prefer to only have b and lights, c and ligths, and separate classes for bc, cc, I can change the definitions, but I don't think that's needed

Reply:
Yeah let's use `bx` if it's really b + (c or light)

Reply:
updated
also colours were chosen at random so let me know if something's wrong with them